docker/build/installers/install_osqp.sh

```diff
@@ -48,7 +48,8 @@ pushd "osqp-${OSQP_VER}"
     cmake .. \
         -DBUILD_SHARED_LIBS=ON \
         -DCMAKE_INSTALL_PREFIX="${SYSROOT_DIR}" \
-        -DCMAKE_BUILD_TYPE=Release
+        -DCMAKE_BUILD_TYPE=Release \
+        -D CTRLC=OFF
```

Comment:
Please remove the space between `-D` and `CTRLC=OFF`, to keep in accord with other installers.
Thanks.